courses/machine_learning/deepdive2/building_production_ml_systems/solutions/1_training_at_scale.ipynb

```diff
@@ -7,12 +7,12 @@
     "# Training at scale with AI Platform Training Service\n",
```

Comment:
<ul><li> clear output</li><li>"rm -rf ${OUTDIR}" -&gt; "test -d $OUTDIR &amp;&amp; rm -rf $OUTDIR" (we remove the dir only if it exists to avoid a crash when not).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>ok. cool!</p><p> </p>